README.md

```diff
@@ -73,6 +73,14 @@ In development and testing, to update a package update it in requirements.in and
 
 To remove a dependency, remove it from requirements.in, run pip-compile and then pip-sync. If everything works as expected, commit the changes.
 
+### Installation issue on Mac
+You need to have PostgreSQL installed or installing the packages from `requirements.txt` will fail. You can install the database with `brew install postgresql`
```

Comment:
Yeah that makes sense 👍 

I tweaked the README like you said and I also moved `Requirement` section to be under `Installation` instead of `Testing`.